src/main/js/dynamic/DistributionConfiguration.js

```diff
@@ -182,7 +198,7 @@ class DistributionConfiguration extends Component {
         const displayTest = !currentChannel.readOnly && DescriptorUtilities.isOperationAssigned(currentChannel, OPERATIONS.EXECUTE);
         const displaySave = !currentChannel.readOnly && DescriptorUtilities.isOneOperationAssigned(currentChannel, [OPERATIONS.CREATE, OPERATIONS.WRITE]);
         const isReadOnly = currentChannel.readOnly;
-
+        const channelDescriptorName = channelConfig ? channelConfig.descriptorName : null;
```

Comment:
I believe this is equal to `const channelDescriptorName = channelConfig && channelConfig.descriptorName`

Reply:
I will update it that should work.